pkgs/tools/system/ansible/default.nix

```diff
@@ -19,14 +19,10 @@ pythonPackages.buildPythonPackage rec {
   dontPatchELF = true;
   dontPatchShebangs = true;
 
-  propagatedBuildInputs = with pythonPackages; [
+  pythonPath = with pythonPackages; [
```

Comment:
The inputs need to be propagated though; they're not just used at build time. They just don't need to be exposed to downstream dependencies. Perhaps a switch such as `isLibrary = false;` so that if someone wanted to use ansible as a library in python code they could do that (and they pythonpaths would remain). The thing is that if one just wants to use ansible as an executable only, then dragging the pythonpaths from ansible's dependencies into the stdenv causes problems.